Clockwork/DataSource/EloquentDataSource.php

```diff
@@ -257,7 +257,9 @@ protected function createRunnableQuery($query, $bindings, $connection)
 	protected function quoteBinding($binding, $connection)
 	{
 		$connection = $this->databaseManager->connection($connection);
-
+		if (get_class($connection) === 'DesignMyNight\Elasticsearch\Connection') {
```

Comment:
I was also thinking that below condition might be better, but our specific case relies on 'DesignMyNight\Elasticsearch\Connection' 

```php
if (!$connection->getPdo())
```